### ISSUE TITLE
Initial learning rate in configuration

### ISSUE THREAD

**Issue by lindsey98** [euwern/proxynca_pp#12]
Hi,
  May I ask why you set the initial learning rate for "proxynca" to be 4e2, shouldnt it be 4e-2?   

```
 "opt": {
        "type": "torch.optim.Adam",
        "args": {
            "embedding": {
                "weight_decay": "0"
            },
            "backbone": {
                "weight_decay": "0"
            },
            "proxynca": {
                "weight_decay": "0", 
                "lr": "4e2"
            },
```
